meson.build

```diff
@@ -331,6 +333,7 @@ gnome_svg_files = files(
 	'data/gnome/logitech-g9.svg',
 	'data/gnome/logitech-g900.svg',
 	'data/gnome/roccat-kone-xtd.svg',
+	'data/gnome/steelseries-rival310.svg',
```

Comment:
the svg still looks a lot darker than all the other ones. not a killer but it makes the contrast to the LED symbol worse. did you use the same color that the other mice use for the main body? maybe it's github's SVG rendering that's different here.